src/gtest/test_sidechain_to_mempool.cpp

```diff
@@ -449,6 +450,49 @@ TEST_F(SidechainsInMempoolTestSuite, FwdsAndCertInMempool_FwtRemovalDoesNotAffec
     EXPECT_TRUE(mempool.mapSidechains.at(scId).backwardCertificate == cert.GetHash());
 }
 
+TEST_F(SidechainsInMempoolTestSuite, ImmatureExpenditureRemoval) {
+    //Create and mature coinbase
```

Comment:
Edit this to `Create coinbase`, since maturing has a different comment below

Reply:
done